.github/workflows/linux.yml

```diff
@@ -47,15 +48,16 @@ jobs:
         python tools/linter.py --branch origin/${{ github.base_ref }}
```

Comment:
Looks like this is broken by the change.

Reply:
I am unsure why this breaks – but I can revert for now, the `lint` job overall should be pretty fast

Reply:
I reverted in https://github.com/numpy/numpy/pull/25900/commits/9858434de6ad320e8ce1ce05bce583768e0d030e. I think the reason why it didn't work was because the action did fetch the tags, but with unitary depth, i.e., just the one commit.